viewing_party/party.py

```diff
@@ -0,0 +1,259 @@
+#Wave_One part 1 
+def create_movie (title, genre, rating):
+    
+    if title and genre and rating:
+        movie_dict = {
+        "title": title, 
+        "genre": genre, 
+        "rating": rating }
+        return movie_dict 
+    else:
+        return None
+
+#Wave_One part 2 
+def add_to_watched(user_data, movie): 
+    user_data["watched"].append(movie)
+    return user_data
+
+
+#Wave_One part 3 
+
+def add_to_watchlist(user_data, movie):
+
+    user_data["watchlist"].append(movie)
+    return user_data
+
+
+#Wave_One part 4 
+
+def watch_movie (user_data, title): 
+
+    for movie in user_data["watchlist"]:
+        if movie["title"] == title:
+
+            user_data["watched"].append(movie)
+            user_data["watchlist"].remove(movie)
```

Comment:
even though remove works great here, be careful using it in a for loop. It can cause side effects. Here is a link that explains it more with examples https://thispointer.com/python-remove-elements-from-a-list-while-iterating/